front/public/swagger.json

```diff
@@ -717,7 +717,7 @@
                   },
                   "blocking": {
                     "type": "boolean",
-                    "description": "Whether to wait for the agent to generate the initial message",
+                    "description": "Whether to wait for the agent to generate the initial message (if blocking = false, you will need to use streaming events to get the messages)",
                     "default": false,
```

Comment:
Shouldn't this be `"default" : true` to be aligned with the above?

Reply:
Hehe yeah changed thx

Reply:
All good now 😇